.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+## Proposed changes
+
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. -->
+
+## Release Notes
+<!-- Include the Milestone Number and a small description of your change that will be added to the changelog -->
+
+Milestone:
+
+Changelog:
+
+## Types of changes
+
+What types of changes does your code introduce to Zowe Explorer?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
```

Comment:
Do we want more generic wording for the last option? Perhaps "Updates to Documentation or Tests", as we have a number of PRs open currently that only touch unit test files.